graphql_compiler/tests/test_ir_generation_errors.py

```diff
@@ -490,13 +490,20 @@ def test_tag_directive_constraints(self):
             }
         }''')
 
+        unused_tag = (GraphQLCompilationError, '''{
+            Animal {
+                name @tag(tag_name: "name")
+            }
+        }''')
+
         errors_and_inputs = (
```

Comment:
Should I add filters to the other tag constraint tests? This change makes it a bit hard to isolate problems (e.g. the filter might have to filter on a tag on a vertex field or with an illegally named argument)?

Reply:
Yes, please do.